packages/dd-trace/src/proxy.js

```diff
@@ -76,6 +76,10 @@ class Tracer extends NoopProxy {
     this._pluginManager.configurePlugin(...arguments)
     return this
   }
+
+  get TracerProvider () {
```

Comment:
This needs to be added to our API docs.

Reply:
Just `docs/API.md`? I'm unclear how to add things to the generated part of the docs.

Reply:
Ah, that's from the `index.d.ts` file...which I guess this needs type definitions too. 🤦🏻